staging/src/k8s.io/kube-scheduler/config/v1alpha2/types_pluginargs.go

```diff
@@ -108,6 +110,13 @@ type UtilizationShapePoint struct {
 	Score int32
 }
 
+// UnmarshalJSON provides case insensitive unmarshalling for the type.
+// TODO remove when copying to v1beta1.
+func (t *UtilizationShapePoint) UnmarshalJSON(data []byte) error {
+	type internal *UtilizationShapePoint
+	return gojson.Unmarshal(data, internal(t))
+}
```

Comment:
is a test needed to demonstrate this works?

Reply:
They are in scheme_test.go